eplaunch/interface/frame.py

```diff
@@ -336,7 +336,7 @@ def run_workflow(self):
                 self.status_bar.SetStatusText('Starting workflow', i=0)
                 self.workflow_worker = WorkflowThread(
                     self, self.current_workflow, self.directory_name, self.current_file_name,
-                    {'weather': self.current_weather_file}
+                    {'weather': self.current_weather_file, 'workflow location': self.current_workflow_directory}
```

Comment:
@Myoldmopar I am passing the workflow directory now to the workflows. This should allow them to find their exe's without requiring cmake tokens.